bci_tester/data.py

```diff
@@ -735,6 +735,46 @@ def create_BCI(
     for ver, tag in (("15.6", "0.21"),)
 ]
 
+PROMETHEUS_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/prometheus:{tag}",
+        bci_type=ImageType.APPLICATION,
+        forwarded_ports=[PortForwarding(container_port=9090)],
+        available_versions=versions,
+    )
+    for versions, tag in ((("15.6", "tumbleweed"), "2"),)
+]
+
+ALERTMANAGER_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/alertmanager:{tag}",
+        bci_type=ImageType.APPLICATION,
+        forwarded_ports=[PortForwarding(container_port=9093)],
+        available_versions=versions,
+    )
+    for versions, tag in ((("15.6", "tumbleweed"), "0.26"),)
+]
+
+BLACKBOX_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/blackbox_exporter:{tag}",
+        bci_type=ImageType.APPLICATION,
+        forwarded_ports=[PortForwarding(container_port=9115)],
+        available_versions=versions,
+    )
+    for versions, tag in ((("15.6", "tumbleweed"), "0.24"),)
+]
+
+GRAFANA_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/grafana:{tag}",
+        bci_type=ImageType.APPLICATION,
+        forwarded_ports=[PortForwarding(container_port=3000)],
+        available_versions=versions,
+    )
+    for versions, tag in ((("15.6", "tumbleweed"), "9"),)
```

Comment:
We have grafana 10 in TW, so you'll have to do something like:
```suggestion
    for versions, tag in ((("15.6",), "9"),  (("tumbleweed",), "10"))
```